src/components/molecules/BuySellCoin/BuySellCoin.jsx

```diff
@@ -4,6 +4,38 @@ import { Input } from "antd";
 import "./_BuySellCoin.scss";
 import { useAppProvider } from "../../../context/AppProvider";
 import { TRANSACTION_VALIDITY } from "../../../utils/constants";
+import { useWSCProvider } from "milkomeda-wsc-ui-test-beta";
+import { stringToBigNumber } from "../../../utils/helpers";
```

Comment:
[eslint] 
src/components/molecules/BuySellCoin/BuySellCoin.jsx
  Line 8:10:  'stringToBigNumber' is defined but never used                  no-unused-vars
  Line 14:7:  'cardanoStablecoinAddress' is assigned a value but never used  no-unused-vars